cmake/MLIRUtils.cmake

```diff
@@ -0,0 +1,34 @@
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2022 Advanced Micro Devices, Inc.
+
+function(mlir_gen_iface prefix iface kind)
```

Comment:
I know that I was the author of these shortcuts, but I suggest that you reconsider their use. I like to use them because the enforce a fixed naming scheme for files and generated code elements.

Considering that the rest of xten uses a different style, it might be prudent to not introduce a new one. But that is just my opinion.

Reply:
I think you are right. I removed them.